containerimages/container-disk-images/create-push-vm-image.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+set -ex
+
+SCRIPT_DIR="$(
+    cd "$(dirname "$BASH_SOURCE[0]")"
+    pwd
+)"
+
+export REGISTRY=${REGISTRY:-quay.io}
+export REPOSITORY=${REPOSITORY:-omergi}
```

Comment:
i would change the default value to ? (which enforce to give a value, or some common repo)

Reply:
I think kubevirt registry should be the default, what you think?

Reply:
sounds good if the end official result that prow will create is there